LoopBackTests/LBUserTests.m

```diff
@@ -126,6 +126,16 @@ - (void)testLoginLogout {
     ASYNC_TEST_END
 }
 
+- (void)testResetPassword {
```

Comment:
Would you mind to add this new test to `+ (id)defaultTestSuite` (Line 62)?  Without it, the unit test won't get executed when running the tests automatically.